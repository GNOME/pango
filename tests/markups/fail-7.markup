<span weight="bold" weight="black"> text</span>
