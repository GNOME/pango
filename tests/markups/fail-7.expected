ERROR: Attribute 'weight' occurs twice on <span> tag on line 1 char 44, may only occur once